### PR TITLE
Migrate sscs to v2

### PR DESCRIPTION
**Pull request by adusumillipraveen**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/sscs/aat/01/kustomization.yaml [hunk 1]
```diff
@@ -2,4 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
+  - ../../sscs-case-loader/sscs-case-loader.yaml
```
Comment (thomast1906) [inline apps/sscs/aat/01/kustomization.yaml:5]: Do we not want to include this in aat base rather than 00 & 01?

Comment (adusumillipraveen) [reply apps/sscs/aat/01/kustomization.yaml:5]: scripting is easy this way :D Problem is not all apps have them on both clusters.

Comment (adusumillipraveen) [reply apps/sscs/aat/01/kustomization.yaml:5]: I can try and find a way, if u think it will be cleaner.

File: apps/sscs/sscs-bulk-scan/prod.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: helm.fluxcd.io/v1
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
   name: sscs-bulk-scan
```
Comment (chris-pateman) [inline apps/sscs/sscs-bulk-scan/prod.yaml:4]: If these have no parameters in them, do we need to include them?

Comment (adusumillipraveen) [reply apps/sscs/sscs-bulk-scan/prod.yaml:4]: Not needed, but not easy to make the script aware of this and remove patches.Its just lift and shift of existing config, minimum needed for migration.

File: apps/sscs/aat/01/kustomization.yaml (hunk 1 above)
Comment (thomast1906) [reply apps/sscs/aat/01/kustomization.yaml]: Cheers! Sorry just saw the duplicate of that base reference^

**Review by thomast1906 — APPROVED** [review]
QQ - do we want to move `../../sscs-hearings-api/sscs-hearings-api.yaml `& `sscs-bulk-scan.yam`l to base to avoid duplication ?

**Pull request merged**